Manpager: apply s commands with nogdefault

toxeus opened the pull request:
Problem: The `s` command with `g` flag only substitutes
         one occurrence when `gdefault` is set.<br>
Solution: Unset `gdefault`.

chrisbra commented on runtime/plugin/manpager.vim:
```diff
@@ -30,10 +30,10 @@ function s:ManPager()
 
   " Remove ansi sequences
-  silent! keepj keepp %s/\v\e\[%(%(\d;)?\d{1,2})?[mK]//ge
+  silent! 'keepj keepp %s/\v\e\[%(%(\d;)?\d{1,2})?[mK]//e' .. (&gdefault ? '' : 'g')
```
same here: wrap into an exe call

toxeus committed:
Manpager: apply g flag conditionally to s command

Problem: The `s` command with `g` flag only substitutes
         one occurrence when `gdefault` is set.
Solution: Use `g` flag conditionally.
